server/datastore/mysql/android_test.go

```diff
@@ -8,12 +8,15 @@ import (
 	"github.com/fleetdm/fleet/v4/server/fleet"
 	"github.com/fleetdm/fleet/v4/server/mdm/android"
 	"github.com/fleetdm/fleet/v4/server/ptr"
+	"github.com/fleetdm/fleet/v4/server/test"
+	"github.com/jmoiron/sqlx"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestAndroid(t *testing.T) {
 	ds := CreateMySQLDS(t)
+	TruncateTables(t, ds)
```

Comment:
This was needed because the initial state of labels is not quite the same before and after truncation. Doing it here ensures we don't need to rely on an ordering of test cases.